BUILD-INSTALL.md

```diff
@@ -0,0 +1,138 @@
+Build and Install KeePassXC
```

Comment:
I think this file should still be called INSTALL.md. It's quite a standard name for many software packages.

Reply:
I see your point. I've updated my PR to change the name back. Thanks.